modules/extdata/uri/pom.xml

```diff
@@ -407,6 +430,7 @@
                     </execution>
                 </executions>
             </plugin>
+
```

Comment:
This empty line is excess, closing tag should go one after another without empty lines. Only similar tags and sections are separated by new line in pom.xml